pkg/nodes/node.go

```diff
@@ -23,10 +25,75 @@ const (
 
 // Builder provides struct for Node object containing connection to the cluster and the list of Node definitions.
 type Builder struct {
-	Definition *v1.Node
-	Object     *v1.Node
-	apiClient  *clients.Settings
-	errorMsg   string
+	Definition  *v1.Node
+	Object      *v1.Node
+	apiClient   *clients.Settings
+	errorMsg    string
+	drainHelper *drain.Helper
+}
+
+// SetDrainHelper builds drain Helper that contains parameters to control the behaviour of drain.
+func (builder *Builder) SetDrainHelper(
+	force bool,
+	ignoreDaemonsets bool,
+	deleteLocalData bool,
+	gracePeriod int,
+	skipWaitForDeleteTimeoutSeconds int,
+	timeout time.Duration,
+) *Builder {
```

Comment:
Please remove *Builder from function return. No need for that. This method just sets **builder.drainHelper**

Reply:
updated, thanks